docs/pages/reference/terraform/data-sources/access_list.mdx

```diff
@@ -0,0 +1,168 @@
+---
+title: "teleport_access_list Data Source - teleport"
+description: |-
+  
+---
+
+# teleport_access_list (Data Source)
+
+
```

Comment:
Is there a way to add a paragraph here for each resource that indicates what the resource is/does? For resources that we talk about a lot in the docs, like roles, this probably isn't as useful, but there are some more obscure resources (e.g., `cluster_networking_config`) that could use a brief description.

Reply:
Yes, we have 2 ways of doing this:
- via the TF resource description. Currently the Terraform provider does not have the resource description in iots schema. I'll see if I can edit https://github.com/gravitational/protoc-gen-terraform so it picks up the message description and adds in the schema.
- via per-resource template overrides. All resources share the same default documentation template, but if we want to add more content or examples in a specific resource we can drop a file in `integrations/terraform/templates/{resource|datasource}/` and the doc generator will use it instead. I will document how to contribute to the terraform reference. We discussed adding examples for common use-cases with @thedevelopnik and I think that would be a good place to do it.